Classes/Event/EventTransport.php

```diff
@@ -1,89 +0,0 @@
-<?php
-namespace Neos\Cqrs\Event;
-
-/*
- * This file is part of the Neos.Cqrs package.
- *
- * (c) Contributors of the Neos Project - www.neos.io
- *
- * This package is Open Source Software. For the full copyright and license
- * information, please view the LICENSE file which was distributed with this
- * source code.
- */
-
-use Neos\Cqrs\Message\MessageMetadata;
-use TYPO3\Flow\Annotations as Flow;
-
-/**
- * The EventTransport is a wrapper container for a domain event that is enriched with metadata.
- * This class is immutable, if the event contained is implemented immutable.
- *
- * @Flow\Proxy(false)
- */
-class EventTransport
```

Comment:
Not sure about the new name EventWithMetadata is basically a technical description of what the object IS

EventTransport is the description of what the object DO ... and personally prefer this way of thinking. 

Especially if you think about future refactoring (you know the unplanable parts of our daily work), what if at some point this objet need to transport more that one event with metadata, did we rename the object ?